deployment/README.md

```diff
@@ -18,13 +18,10 @@ Deploying your App on the Cloud
      - [App deployment using spark-submit](#using-spark-submit-2)
 
 # Pre-requisites:
-1. Clone and successfully build [Spark .NET](https://github.com/dotnet) by following the [Quick Start instructions](https://github.com/dotnet/spark#quick-start-tldr).
-2. Download and install [.NET Core](https://dotnet.microsoft.com/download) <span style="color: red">2.1+</span> for your operating system.
-3. Tool for creating a `tgz` file: `tar` on Linux, [7-ZIP](https://www.7-zip.org/) on Windows, etc.
-4. Tool to copy files to a distributed file system.
+1. Tool to copy files to a distributed file system.
    - ADLS, WASB &rarr; [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/)
    - S3 &rarr; [AWS CLI](https://aws.amazon.com/cli/)
-5. Download [install-worker.sh](install-worker.sh) to your local machine. This is a helper script that we will use later in the installation section to copy Spark .NET dependent files into your Spark cluster's worker nodes. install-worker.sh takes in three parameters:
+2. Download [install-worker.sh](install-worker.sh) to your local machine. This is a helper script that we will use later in the installation section to copy Spark .NET dependent files into your Spark cluster's worker nodes. install-worker.sh takes in three parameters:
    1. The Cloud Provider: `azure` or `aws`
    2. URI where `worker.tgz` is uploaded.
```

Comment:
You don't describe what `worker.tgz` is or where to obtain it from. I suggest renaming things to be more consistent across the document. Either say, download the release and rename to worker.tgz or just put the release name throughout. Whichever is easier.

Reply:
Another way, you could word it is:

URI where `worker.tgz` is uploaded (see #section for instructions on where to download this)

OR

URI where Microsoft.Spark.Worker.tgz is uploaded (see #section for instructions on where to download this) - this also indicates this is a Microsoft zip file.

Also, the release is tar.gz whereas the wording here is tgz. Can you update?